pkg/didcomm/protocol/didexchange/models.go

```diff
@@ -74,6 +74,11 @@ type Request struct {
 	Label      string            `json:"label,omitempty"`
 	Connection *Connection       `json:"connection,omitempty"`
 	Thread     *decorator.Thread `json:"~thread,omitempty"`
+	// DID the did of the requester. Optional, may be present within `connection` instead.
```

Comment:
@llorllale @Moopli - are these properties supposed to be optional or is this for interop?

Reply:
https://github.com/hyperledger/aries-rfcs/pull/608